gradle.properties

```diff
@@ -1,7 +1,7 @@
 kotlin.code.style=official
 
 # codegen
-smithyVersion=1.22.0
+smithyVersion=[1.25.0,1.26.0[
```

Comment:
Is this correct to have two opening brackets?

Reply:
Yes, this is correct.

Reply:
https://docs.gradle.org/current/userguide/single_versions.html

Reply:
lol I need to get smart on Gradle